packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -57,11 +58,28 @@ export interface ExpansionHub extends RevHub {
     getFTDIResetControl(): Promise<boolean>;
 
     // DIO
-    setDigitalSingleOutput(dioPin: number, value?: boolean): Promise<void>;
+    setDigitalSingleOutput(digitalChannel: number, value: DigitalState): Promise<void>;
     setDigitalAllOutputs(bitPackedField: number): Promise<void>;
+
+    /**
+     * Set a digital pin as input or output.
+     * @param dioPin
+     * @param direction
+     */
     setDigitalDirection(dioPin: number, direction: DIODirection): Promise<void>;
     getDigitalDirection(dioPin: number): Promise<DIODirection>;
-    getDigitalSingleInput(dioPin: number): Promise<boolean>;
+
+    /**
+     * Read the state of a given digital pin.
+     * Throws an error if the pin is not configured for input
+     *
+     * @param dioPin
+     */
+    getDigitalSingleInput(dioPin: number): Promise<DigitalState>;
```

Comment:
`dioPin` needs to be `digitalChannel`

Reply:
I think a better name would be `getDigitalInput()` (`getDigitalAllInputs() would become `getAllDigitalInputs()`). Outputs would follow the same scheme.

Reply:
Renamed the methods.